jscomp/ml/location.mli

```diff
@@ -69,6 +69,8 @@ val printer : (formatter -> t -> unit) ref
 val warning_printer : (t -> formatter -> Warnings.t -> unit) ref
 (** Hook for intercepting warnings. *)
 
+val formatter_for_warnings : formatter ref
```

Comment:
unfortunately i need this interface, otherwise I can't reset the warning buffer between multiple runs